ksp_plugin/plugin.cpp

```diff
@@ -1060,6 +1060,14 @@ not_null<Vessel*> Plugin::GetVessel(GUID const& vessel_guid) const {
   return FindOrDie(vessels_, vessel_guid).get();
 }
 
+void Plugin::ClearOrbitAnalysersOfVesselsOtherThan(Vessel const& vessel) {
+  for (auto const& [guid, v] : vessels_) {
+    if (v.get() != &vessel) {
```

Comment:
Let's not depend on pointers: `guid != vessel.guid()`.

Reply:
There is precedent for that, e.g., https://github.com/mockingbirdnest/Principia/blob/d74f56407e4204d026be17fe8161da428672d6a6/ksp_plugin/plugin.cpp#L476-L477
and it is safe since `Vessel` is neither copiable nor movable.